exporter/otlpexporter/consume_contract_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package otlpexporter
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"go.opentelemetry.io/collector/config/confignet"
+
+	"github.com/cenkalti/backoff/v4"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configgrpc"
+	"go.opentelemetry.io/collector/config/configtls"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+	"go.opentelemetry.io/collector/exporter/exportertest"
+)
+
+const DefaultOTLPPort = 9999
+
+// NewTestRetrySettings returns the default settings for otlp exporter test.
+func NewTestRetrySettings() exporterhelper.RetrySettings {
```

Comment:
There is no need to export this function

Reply:
done